test/deployframework/framework.go

```diff
@@ -61,7 +64,7 @@ type DeployFramework struct {
 }
 
 // New is the constructor function that creates and returns a new DeployFramework object
-func New(logger logrus.FieldLogger, runLocal, runDevSetup bool, nsPrefix, repoDir, kubeconfig string) (*DeployFramework, error) {
+func New(logger logrus.FieldLogger, runLocal, runDevSetup bool, nsPrefix, repoDir, repoVersion, kubeconfig string) (*DeployFramework, error) {
```

Comment:
I think we should hold off on renaming this constructor as instantiating a new framework is done by calling `deployframework.New(...)` which is the correct usage (as deployframework essentially acts a namespace). If this was a method, instead of a function I would agree that it should be changed to something like `NewDeployerFramework`.

Reply:
ah okay